rest-api-spec/src/main/resources/rest-api-spec/api/connector.update_scheduling.json

```
@@ -0,0 +1,39 @@
+{
+  "connector.update_scheduling": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/enterprise-search/current/connectors.html",
+      "description": "Updates the scheduling field in the connector document."
+    },
+    "stability": "experimental",
+    "visibility": "feature_flag",
+    "feature_flag": "es.connector_api_feature_flag_enabled",
+    "headers": {
+      "accept": [
+        "application/json"
+      ],
+      "content_type": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_connector/{connector_id}/scheduling",
```

I'd _say_ that actions done on a resource in ES tend to have underscore:

```suggestion
          "path": "/_connector/{connector_id}/_scheduling",
```

I think that's more in line with what we have. WDYT @kderusso ? (Or anyone that can keep me honest here)